dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/util/json/DatasetFieldParser.java

```diff
@@ -112,11 +119,13 @@ private JsonArrayBuilder parseChildren(boolean excludeEmailFields, DatasetField
             childObject.add("typeClass", typeClassString(childType));
 
             if (childType.isControlledVocabulary()) {
-                for (ControlledVocabularyValue cvv
-                        : sort(dsfChild.getControlledVocabularyValues(),
-                               ControlledVocabularyValue.DisplayOrder)) {
```

Comment:
One question: here we have sorting of elements, and in the new version we don't – is this ok, that we're not doing sort anymore?

Reply:
Right now there is no multiple elements support so it doesn't matter. But in case in the future (I doubt it) we will support it, I think I will revert this change.